python/lsst/ts/MTDome/encoding_tools.py

```diff
@@ -19,7 +19,57 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+import logging
 import json
+import os
+
+import jsonschema
+
+# Load the JSON schemas
+script_dir = os.path.dirname(__file__)
+command_schema = json.loads(
+    open(os.path.join(script_dir, "schemas/command.jschema")).read()
+)
+response_schema = json.loads(
+    open(os.path.join(script_dir, "schemas/response.jschema")).read()
+)
+amcs_status_schema = json.loads(
+    open(os.path.join(script_dir, "schemas/amcs_status.jschema")).read()
+)
+apscs_status_schema = json.loads(
+    open(os.path.join(script_dir, "schemas/apscs_status.jschema")).read()
+)
+lcs_status_schema = json.loads(
+    open(os.path.join(script_dir, "schemas/lcs_status.jschema")).read()
+)
+lwscs_status_schema = json.loads(
+    open(os.path.join(script_dir, "schemas/lwscs_status.jschema")).read()
+)
+moncs_status_schema = json.loads(
+    open(os.path.join(script_dir, "schemas/moncs_status.jschema")).read()
+)
+thcs_status_schema = json.loads(
+    open(os.path.join(script_dir, "schemas/thcs_status.jschema")).read()
+)
+
+# dict to help look up the schema to use for validation
+schemas = {
```

Comment:
This can be a bit improved - the advantage would be that you can get easily error handling if needed. Instead of _schema = json.loads(...), do:

```python
def open_schema(name):
    return json.loads(open(os.path.join(script_dir, "schemas", filename)).read())
```

and then:

```python
   "command": open_schema("command.jschema")
```

etc.